package.json

```diff
@@ -175,7 +175,7 @@
         "deepmerge": "^2.1.1",
         "draft-js": "^0.10.1",
         "enzyme": "3.8.0",
-        "enzyme-adapter-react-16": "^1.10.0",
+        "enzyme-adapter-react-16": "1.12.1",
```

Comment:
Does this need to be pinned to this specific version?

Reply:
not fixed, can change this to ^